src/api/connector/external/rest.js

```diff
@@ -1,16 +1,101 @@
-/*
-Generic HTTP REST connector.
-This connector should implement OPEN API specification.
-https://github.com/OAI/OpenAPI-Specification
-https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md
-*/
+import http from 'axios';
+import { toLower } from 'lodash';
+import { logger } from '../../../utility';
+
+const getIdentifier = (source, options) => {
+  const identifierName = source.schema.identifier;
+  const identifier = options.params[identifierName];
+
+  if (!identifier) {
+    logger.error('Identifier field not found in params');
+    throw new Error('Identifier field not found in params');
+  }
+
+  return identifier;
+};
+
+const getSortPrefix = (sortBy) => {
+  let sortPrefix = '';
+
+  if (sortBy === 'desc') sortPrefix = '-';
```

Comment:
Do we need casing normalisation here?

Reply:
Casing normalization for order param, e.g. `Desc`? It won't hurt if I add it.

Reply:
Yup. Just use lodash toLower there. :)